src/components/contactInfoForm/ContactInfoForm.tsx

```diff
@@ -1,25 +1,86 @@
-import React from "react";
-import { useForm } from "react-hook-form";
+import React, { useState, useEffect } from "react";
+import { FieldValues, useForm } from "react-hook-form";
 import { yupResolver } from "@hookform/resolvers/yup";
 import i18next from "localization/en/en.json";
+import { toast } from "react-toastify";
+import { useSelector } from "react-redux";
 import LayoutElementWithTitle from "components/layoutElementWithTitle/LayoutElementWithTitle";
 import propsDataCollection from "components/contactInfoForm/staticData";
 import FormSubmitButton from "components/UI/buttons/formSubmitButton/FormSubmitButton";
 import { SButtonWrapper } from "components/profileEditForm/styles";
 import contactEditFormValidationSchema from "validation/contactEditFormValidationSchema";
+import { IContactInfoState } from "components/contactInfoForm/typesDef";
+import {
+  useGetUserQuery,
+  useUpdateUserMutation,
+} from "services/contactInfo/contactInfoAPI";
+import { RootState } from "store/store";
 
 function ContactInfoForm() {
+  const [initialState, setInitialState] = useState<IContactInfoState>({
+    email: "",
+    firstName: "",
+    lastName: "",
+    phone: null,
+  });
```

Comment:
can we use null instead?

Reply:
no I mean can we use null as the default state, without this list on empty lines

Reply:
fixed